ui/app/templates/vault/cluster/clients/index.hbs

```diff
@@ -7,6 +7,7 @@
 </PageHeader>
 
 <div class="tabs-container box is-bottomless is-marginless is-fullwidth is-paddingless" data-test-pricing-metrics>
+  {{! template-lint-configure no-unknown-arguments-for-builtin-components "warn" }}
```

Comment:
Why are we changing this to warn?

Reply:
Warn as kind of a reminder. We likely can't do anything about it soon, but it's not something we want to forget. A discussion Jordan and I had.